conf/pipeline/viralrecon/eddie.config

```diff
@@ -0,0 +1,9 @@
+process {
+  withName : '.*PICARD.*' {
```

Comment:
You will also need to edit this file https://github.com/nf-core/configs/blob/master/pipeline/viralrecon.config loading the `eddie` profile like [here](https://github.com/nf-core/configs/blob/47217cae5b2286b7a288ed1bd4616990d5de7a26/pipeline/rnaseq.config#L12)

Reply:
Be great if you can test this via your fork to see if it works. Should just need to change [this](https://github.com/nf-core/viralrecon/blob/f0171324a60d1759ca7f5d02351372d97d22cc12/nextflow.config#L106) parameter.

Reply:
> You will also need to edit this file https://github.com/nf-core/configs/blob/master/pipeline/viralrecon.config loading the `eddie` profile like [here](https://github.com/nf-core/configs/blob/47217cae5b2286b7a288ed1bd4616990d5de7a26/pipeline/rnaseq.config#L12)

Done.